commands/other/8ball.js

```diff
@@ -0,0 +1,38 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class EightBallCommand extends Command {
+  constructor(client) {
+    super(client, {
+      name: '8ball',
+      aliases: ['eightball'],
+      memberName: '8Ball',
+      group: 'other',
+      description: 'Get the answer to anything!',
+      args: [
+        {
+          key: 'text',
+          prompt: 'What do you want to ask?',
+          type: 'string'
+        }
+      ]
+    });
+  }
+
+  run(message) {
+        const ballanswers = fs.readFileSync( 
+      'resources/fun/8ball.json',
```

Comment:
location doesn't exist  maybe 'resources/other/8ball.json like in your pr

Reply:
spot on again